src/lib/modules/movement.js

```diff
@@ -4,135 +4,181 @@
 //
 // * * * * * * * * * * * * * * * * * * * * * * * * * * *
 
-import { writable, get } from "svelte/store"
-import inRange from "lodash/inRange.js"
-import _ from 'lodash'
-import throttle from 'lodash/throttle.js'
-import { uiState } from '$lib/modules/ui.js'
-import { players, moveTo } from "$lib/modules/engine.js"
-import { GRID_SIZE, localPlayer } from "$lib/modules/world.js"
+import { writable, get } from "svelte/store";
+import inRange from "lodash/inRange.js";
+import _ from "lodash";
+import throttle from "lodash/throttle.js";
+import { uiState } from "$lib/modules/ui.js";
+import { players, moveTo } from "$lib/modules/engine.js";
+import { GRID_SIZE, localPlayer } from "$lib/modules/world.js";
 
 // *** VARIABLES
-export const roomIntent = writable(false)
-export const objectIntent = writable(false)
-export const activeZone = writable(false)
-export const currentRoom = writable(false)
-export const centeringInlineStyles = writable("transform: translateX(-50%) translateY(-50%);")
+export const roomIntent = writable(false);
```

Comment:
This is an automatic linting by the editor. The important part is the `moveToPlayerPosition` function.